gradio/components/video.py

```diff
@@ -239,6 +239,8 @@ def preprocess(
             return output_file_name
         elif not self.include_audio:
             output_file_name = str(file_name.with_name(f"muted_{file_name.name}"))
+            if Path(output_file_name).exists():
```

Comment:
Can you please add a unit test @breengles ? Otherwise looks good!

Reply:
Hi @freddyaboulton! I added a simple test just like the one for the default behavior of `gr.Video()`